actions/validate-payment/index.js

```diff
@@ -29,13 +29,10 @@ async function main(params) {
       return webhookErrorResponse(`Failed to verify the webhook signature: ${error}`);
     }
 
-    let payload = params;
-    if (params.__ow_body) {
-      // in the case when "raw-http: true" the body needs to be decoded and converted to JSON
-      payload = JSON.parse(atob(params.__ow_body));
```

Comment:
Actually in this line, payload does not always have the same meaning, can be param or body. Changed it to use the same code base as `filter-payment`, which explicitly indicates and is always body.